server/provider/networkProvider.go

```diff
@@ -11,6 +11,7 @@ import (
 	"github.com/ElrondNetwork/elrond-go-core/hashing"
 	"github.com/ElrondNetwork/elrond-go-core/marshal"
 	logger "github.com/ElrondNetwork/elrond-go-logger"
+	"github.com/ElrondNetwork/elrond-go/storage/lrucache"
```

Comment:
can we reference here elrond-go-storage instead?
so we remove the dependency to elrond-go

Reply:
Indeed, fixed! 

However, we cannot fully remove dependency on _elrond-go_, at this moment (there are several subpackages of _elrond-go_ that are still used by _rosetta_).